unit/ktest.c

```diff
@@ -280,17 +283,18 @@ static void register_repeated_attempt_test(void)
 	size = do_request(&kmsg, plen, &kresp2);
 
 	/* Response consistency */
-	g_assert(size == sizeof(kresp2.cred));
+	assert(size == sizeof(kresp2.cred));
 
 	/* Response opcode & result */
-	g_assert(kresp2.hdr.type == KNOT_MSG_REG_RSP);
-	g_assert(kresp2.action.result == 0);
-	g_assert_cmpmem(&kresp, size, &kresp2, size);
+	assert(kresp2.hdr.type == KNOT_MSG_REG_RSP);
+	assert(kresp2.action.result == 0);
+	ret = memcmp(&kresp, &kresp2, size);
+	assert(ret == 0);
 
-	g_message("UUID: %.36s token:%.40s\n", kresp2.cred.uuid, kresp2.cred.token);
+	printf("UUID: %.36s token:%.40s\n", kresp2.cred.uuid, kresp2.cred.token);
 }
 
-static void register_new_id(void)
+static void register_new_id(const void *test_data)
```

Comment:
Any specific reason for it not to be `static void register_new_id_test(const void *test_data)`?
Ending wit *_test* just like the similar functions